services/galley/src/Galley/Cassandra/TeamFeatures.hs

```diff
@@ -73,13 +73,13 @@ getApplockFeatureStatus tid = do
   let q = query1 select (params LocalQuorum (Identity tid))
   mTuple <- retry x1 q
   pure $
-    mTuple >>= \(mbStatusValue, mbEnforce, mbTimeout) ->
-      TeamFeatureStatusWithConfig <$> mbStatusValue <*> (TeamFeatureAppLockConfig <$> mbEnforce <*> mbTimeout)
+    mTuple >>= \(mbStatusValue, mbEnforce, mbTimeout, mbPaymentStatusValue) ->
+      TeamFeatureStatusWithConfig <$> mbStatusValue <*> (TeamFeatureAppLockConfig <$> mbEnforce <*> mbTimeout) <*> Just mbPaymentStatusValue
```

Comment:
Does this do the right thing for `null` values?  I don't think it does, since the `<*>` catches the `Just` you've added, and not the one from within the `mbPaymentStatusValue`.

Reply:
This is also resolved by the with/without payment status change.